browser-tests/registrationView.ts

```diff
@@ -0,0 +1,95 @@
+import { Selector } from 'testcafe';
+
+import { login } from './utils/login';
+import { testURL } from './utils/settings';
+import { registrationFormSelector } from './pages/registrationFormSelector';
+import { sentProfileSelector } from './pages/sentProfileSelector';
+
+fixture('Test registration form').page(testURL());
+
+test('Test all required fields if user is adult', async t => {
+  await login(t, 'adult');
+
+  await t.click(registrationFormSelector.submitButton);
+
+  const allRequiredErrors = await Selector(
+    'div[class^="TextInput-module_helperText"]'
+  ).count;
+
+  await t.expect(allRequiredErrors).eql(6);
+});
+
+test('Test all required fields if user is minor', async t => {
+  await login(t, 'minor');
+
+  await t.click(registrationFormSelector.submitButton);
+
+  const allRequiredErrors = await Selector(
+    'div[class^="TextInput-module_helperText"]'
+  ).count;
+
+  await t.expect(allRequiredErrors).eql(10);
```

Comment:
Minor comment. Could even a similar check to `errorCount > 0` be enough here? This looks good also. I'm thinking about a scenario where we add or remove fields--those would break this test without necessarily breaking the functionality.

Reply:
I guess it could be enough. My main question is do we only wan't to test that user can't submit the form straight up, or do we want to test that correct amount of fields are marked as required, based on user's age.

Reply:
Hmn, really good point 🤔 In that case I would maybe leave it as is.

One approach would be to fill the fields that are normally required, and then observe that the submission still fails. But I'm not really sure if that's better. I'd leave it and make changes later is any problems arise.